src/Math.php

```diff
@@ -16,8 +16,10 @@ protected function bayes(array $lexemes)
         $numerator = 1;
         $denominator = 1;
         foreach ($lexemes as $lexeme) {
-            $numerator *= $lexeme['probability'];
-            $denominator *= 1 - $lexeme['probability'];
+            if ($lexeme['probability'] > 0) {
```

Comment:
if we put this here we make that selected lexeme will not be used in calculating (so we lost important information) and we don't want it